library/rsa.c

```diff
@@ -2470,12 +2485,6 @@ int mbedtls_rsa_rsassa_pkcs1_v15_sign(mbedtls_rsa_context *ctx,
     }
 
     MBEDTLS_MPI_CHK(mbedtls_rsa_private(ctx, f_rng, p_rng, sig, sig_try));
-    MBEDTLS_MPI_CHK(mbedtls_rsa_public(ctx, sig_try, verif));
```

Comment:
This is good but I think this function's cleanup is not complete:
- the comment a few lines above needs updating
- we no longer need `verif`
- I think we don't even need `sig_try` now.